util/nrl_batch_install.sh

```diff
@@ -0,0 +1,404 @@
+#!/usr/bin/env bash
```

Comment:
If it's not too inconvenient, could this go in a subdirectory, say, "nrl/"? I guess in part it depends whether you think there will be other NRL-specific utilities down the road...

Reply:
That's ok with me. How about the site-specific subdirectories `acorn` and `narwhal`? Leave them where they are now?

Reply:
I think so yes, and if more files/directories get added we can always reorganize later (like add top-level directories by org).

Reply:
Done